lib/ansible/modules/system/selinux.py

```diff
@@ -149,79 +150,81 @@ def main():
         module.fail_json(msg='libselinux-python required for this module')
 
     # global vars
-    changed=False
-    msgs                  = []
-    configfile            = module.params['configfile']
-    policy                = module.params['policy']
-    state                 = module.params['state']
-    runtime_enabled       = selinux.is_selinux_enabled()
-    runtime_policy        = selinux.selinux_getpolicytype()[1]
-    runtime_state         = 'disabled'
-    if (runtime_enabled):
+    changed = False
```

Comment:
not a fan of the aligned assignments (is that a pep8 thing? I'm not really a python guy)